docs/tutorial/extensions.md

```diff
@@ -45,6 +45,49 @@ The wrapped function:
 If any of these constraints are violated, the outcome is undefined.
 {% endhint %}
 
+## fhe.multivariate(function)
+
+Allows you to wrap any multivariate function into a table lookup:
+
+```python
+import numpy as np
+from concrete import fhe
+
+def keep(value, keep):
+    return value if keep else np.zeros_like(value, dtype=np.int64)
+
+def multivariate_function(x, y):
+    return fhe.multivariate(keep)(x, y)
```

Comment:
why not
`multivariate_function = fhe.multivariate(keep)`
Can't it be declared top level like usual TLU ?

Reply:
It can be, but I think this is more explicit. If we do that, it'll look like a regular function call.